ConwaysGameOfLife.Tests/LocationShould.cs

```diff
@@ -1,41 +1,13 @@
 using System;
+using System.Collections.Generic;
 using Xunit;
 
 namespace ConwaysGameOfLife.Tests
 {
     public class LocationShould
     {
-        [Fact]
-        public void WhenXAndYAreSame_BeEqual()
-        {
-            var x = 0;
-            var y = 0;
-            var location = new Location(x, y);
-            var one = location;
-            var two = new Location(0, 0);
-
-            var result = (one.Equals(two));
-
-            Assert.True(result);
-        }
-
-        [Fact]
-        public void WhenXAndYAreSame_BeEqualOperator()
-        {
-            var x = 0;
-            var y = 0;
-            var location = new Location(x, y);
-            var one = location;
-            var two = new Location(0, 0);
-
-            var result = (one == two);
-
-            Assert.True(result);
-        }
-
         [Theory]
         [InlineData(0, 0, 0, 0, true)]

```

Comment:
Can we add an extra case here, remove the `expected` param, and `Assert.True` instead
```suggestion
        [InlineData(0, 0, 0, 0)]
        [InlineData(5, 5, 5, 5)]
```

Reply:
Same goes for the `WhenTwoLocationsInstantiated_NotBeEqual` test, but `Assert.False`